Description/en-US.md

```diff
@@ -1,75 +1,74 @@
 # English (US)
+<!-- Since EN-US is supposed to be the "default" description, omit this header when pasting to distribution platforms -->
 
-A Fabric-based modpack for Minecraft that focuses on performance and graphics optimizations. 
+A Fabric-based Minecraft modpack focusing on performance and graphics enhancements.
 
-[![Current review](https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
+[<img src="https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg" alt="Current review" height="140"/> - **a YouTube showcase by Rexus**](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
 
 ### Performance
 
-Fabulously Optimized is usually faster than OptiFine due to the inclusion of [modern performance mods][1] like Sodium and Lithium.
+Fabulously Optimized is usually faster than OptiFine thanks to [modern performance mods][1] such as Sodium, Lithium and more!
+
+### Compatibility
+
+Fabulously Optimized works on any world or server, [supports 5 launchers, is hosted on 3 sites][6] and [translated to many languages][7]. All of its mods don't change vanilla mechanics and are purely visual.
 
 ### OptiFine parity
 
-Fabulously Optimized [supports almost all OptiFine features][2], including shaders, connected textures and zoom. [Get a free cape!][3]
+Fabulously Optimized [supports almost all OptiFine features][2], including shaders, connected textures and zoom. [You can even get your own free cape!][3]
 
 ### Minor tweaks
 
-Fabulously Optimized [includes minor tweaks][4] that don't break the vanilla feel, such as increased chat history, windowed full screen mode and bugfixes to most common issues.
+Fabulously Optimized [includes a few minor tweaks][4] that don't break the vanilla feel, such as increased chat history, windowed fullscreen mode and fixes to most common bugs.
 
 ### Up to date
 
-Fabulously Optimized [receives fast updates][5] and is one of the first modpacks to upgrade to newer Minecraft versions.
+Fabulously Optimized [receives fast updates][5] and is, in fact, one of the first modpacks to upgrade to newer Minecraft versions.
 
-### Works everywhere
+### Built on Fabric
 
-Fabulously Optimized is designed to work on any world or server, [runs on 5 launchers, is hosted on 3 sites][6] and [supports many languages][7].
```

Comment:
Intentional reordering? Elaborate a bit.

Reply:
Yup, I reordered some of those points to flow better into each other.

Reply:
Okay. Reorder it for all languages then please.